.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+src/main/resources/logs/*
```

Comment:
logs is not resources
```suggestion
logs/*
```
![image](https://user-images.githubusercontent.com/20326858/225296865-744bc6f3-96a5-43b8-b3a7-16e329fc8744.png)